src/nni_manager/training_service/kubeflow/kubeflowTrainingService.ts

```diff
@@ -66,6 +68,10 @@ class KubeflowTrainingService implements TrainingService {
     private kubeflowRestServerPort?: number;
     private kubeflowJobPlural?: string;
     private readonly CONTAINER_MOUNT_PATH: string;
+    private azureStorageClient: any;
+    private azureStorageShare: any;
+    private azureStorageSecretName: any;
+    private azureStorageAccountName: any;
```

Comment:
I suggest to define the real type if you know

Reply:
I suggest  you to define TS types in *.d.ts for azure-storage objects, refer src/nni_manager/types for example